Microsoft.Toolkit.Uwp.UI.Controls/QuickReturnHeader/QuickReturnHeader.cs

```diff
@@ -138,67 +171,78 @@ private void QuickReturnHeader_SizeChanged(object sender, SizeChangedEventArgs e
+                    }
                 }
             }
         }
 
         private void StartAnimation()
         {
-            if (_scrollProperties == null)
+            if (scrollProperties == null)
             {
-                _scrollProperties = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(_scrollViewer);
+                scrollProperties = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(scrollViewer);
             }
 
-            var compositor = _scrollProperties.Compositor;
+            var compositor = scrollProperties.Compositor;
 
-            if (_animationProperties == null)
+            if (animationProperties == null)
             {
-                _animationProperties = compositor.CreatePropertySet();
-                _animationProperties.InsertScalar("OffsetY", 0.0f);
+                animationProperties = compositor.CreatePropertySet();
             }
 
-            var expressionAnimation = compositor.CreateExpressionAnimation("Floor(animationProperties.OffsetY - ScrollingProperties.Translation.Y)");
+            previousVerticalScrollOffset = scrollViewer.VerticalOffset;
+
+            animationProperties.InsertScalar("OffsetY", 0.0f);
 
-            expressionAnimation.SetReferenceParameter("ScrollingProperties", _scrollProperties);
-            expressionAnimation.SetReferenceParameter("animationProperties", _animationProperties);
+            ExpressionAnimation expressionAnimation = compositor.CreateExpressionAnimation("Floor(animationProperties.OffsetY - ScrollingProperties.Translation.Y)");
+            expressionAnimation.SetReferenceParameter("ScrollingProperties", scrollProperties);
+            expressionAnimation.SetReferenceParameter("animationProperties", animationProperties);
 
-            _headerVisual = ElementCompositionPreview.GetElementVisual((UIElement)TargetListView.Header);
+            headerVisual = ElementCompositionPreview.GetElementVisual((UIElement)TargetListView.Header);
 
-            if (_headerVisual != null && IsQuickReturnEnabled)
+            if (headerVisual != null && IsQuickReturnEnabled)
             {
-                _headerVisual.StartAnimation("Offset.Y", expressionAnimation);
+                headerVisual.StartAnimation("Offset.Y", expressionAnimation);
             }
         }
 
         private void StopAnimation()
         {
-            if (_headerVisual != null)
+            if (headerVisual != null)
             {
-                _headerVisual.StopAnimation("Offset.Y");
-                _animationProperties.InsertScalar("OffsetY", 0.0f);
+                headerVisual.StopAnimation("Offset.Y");
+                animationProperties.InsertScalar("OffsetY", 0.0f);
 
-                var offset = _headerVisual.Offset;
+                var offset = headerVisual.Offset;
                 offset.Y = 0.0f;
-                _headerVisual.Offset = offset;
+                headerVisual.Offset = offset;
             }
         }
+
```

Comment:
Some code formatting rules are working e.g. "SA1503: Braces must not be omitted". But it seems there are no rules that emit warnings for putting private fields at the top or prefixing them with underscores. Anyway, I fixed these issues now.